src/contracts/BPool.sol

```diff
@@ -231,32 +228,45 @@ contract BPool is BBronze, BToken, BMath {
     _tokens[index] = _tokens[last];
     _records[_tokens[index]].index = index;
     _tokens.pop();
-    _records[token] = Record({bound: false, index: 0, denorm: 0, balance: 0});
+    _records[token] = Record({bound: false, index: 0, denorm: 0});
 
     _pushUnderlying(token, msg.sender, bsub(tokenBalance, tokenExitFee));
     _pushUnderlying(token, _factory, tokenExitFee);
   }
 
+  // NOTE: deprecated method, as balances are calculated on-the-fly
```

Comment:
Could `gulp` just be removed, as we're doing it continuously/implicitly now? Are we expecting people to call this? Nobody is really using V1, so I don't think we need to worry about compatibility with existing pools.

Reply:
we can totally deprecate it 👌 
was thinking that if there's the chance of a contract already developed that does `transfer + gulp`, making `gulp` a no-operation would avoid any revert
but, if there's no current implementation, we can just delete the `gulp` method pointer